src/pages/index.js

```diff
@@ -150,7 +244,15 @@ previewModalCloseButton.addEventListener("click", () => {
   closeModal(previewModal);
 });
 
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again.
* I can't close the `confirmation` modal by clicking on the `close icon` or `cancel` button https://snipboard.io/GA9gMI.jpg 
* When I reload the page, the old `avatar` appears again